components/rhel/rhel-ose/README.rst

```diff
@@ -0,0 +1,53 @@
+Vagrantfile to setup  OpenShift Enterprise
+==========================================
+
+This Vagrantfile sets up the CDK for development with OpenShift.  This file also sets up private networking that will be used to expose various services to the host.  This Vagrantfile also provisions an instance of `OpenShift Enterprise <https://www.openshift.com/enterprise/whats-new.html>`_.
+
+If you are interested in the process used in the Vagranfile to setup OpenShift, please read the comments at the top of the file.
+
+QuickStart
+----------
+
+1. Get latest CDK box (Using Developer Subscription) and add it to vagrant per `Installing the Atomic Developer Bundle <../../../docs/installing.rst>`_.
+
+2. Create a directory for the Vagrant Box
+
+  ``$ mkdir directory && cd directory``
+
+3. Download this Vagrantfile in same directory.
+
```

Comment:
@navidshaikh We are not trying to create documentation for CDK.  As I mentioned before in future we can just point to the relevant documentation when it is published.